data/migrations/V0053__add_ofec_filings_all_mv.sql

```diff
@@ -0,0 +1,481 @@
+
+CREATE OR REPLACE FUNCTION public.get_office_cmte_tp (p_office varchar, p_cmte_tp varchar)
+  RETURNS varchar AS
+$BODY$
+    begin
+
+        return coalesce(p_office, p_cmte_tp);

```

Comment:
Let's only return this in the case that it is H, S or P, that will make the output easier to understand downstream.

Reply:
I did a double check and we have seen in the past, the itemized resources performed better with the idx compound index because we always append the unique index for itemized resources:

https://github.com/18F/openFEC/blob/8e39a1f8e04915ade2e97898b96eda2ea686e906/webservices/common/views.py#L105

The finings endpoint inherits from the ApiResource so it doesn't need it:
 
https://github.com/18F/openFEC/blob/8e39a1f8e04915ade2e97898b96eda2ea686e906/webservices/resources/filings.py#L20

As you mentioned this existed before this PR, but let's simplify the indexes while we are in here. Thanks @fecjjeng and @pkfec!